lib/agenda/now.js

```diff
@@ -8,14 +8,14 @@ const noCallback = require('../no-callback');
  * @function
  * @param {String} name name of job to schedule
  * @param {Object} data data to pass to job
- * @param {Function} cb called when job scheduling fails or passes
+ * @param {Object} mongoOptions MongoDB options to set for job creation
  * @returns {Job} new job instance created
  */
-module.exports = async function(name, data) {
+module.exports = async function(name, data, {session} = {}) {
   // eslint-disable-next-line prefer-rest-params
-  noCallback(arguments, 2);
+  noCallback(arguments, 3);
```

Comment:
What is that supposed to be ? Can you give the reason ? :p

Reply:
It will throw an error when passing the new mongoOptions parameter without changing from 2 to 3.
This change may be considered as breaking change.

Reply:
Yes this is a breaking change.

Reply:
@OmgImAlexis see #1372